### DISCUSSION TITLE
Replacing UnityBootstrapper with PrismBootstrapper

### DISCUSSION THREAD

**Discussion by johnnildo** [PrismLibrary/Prism#2464] [WPF] [answered]
Hello,

As it is well known, `UnityBootstrapper` has been marked obsolete on Prism 7. On Prism 8 it has been removed. 

I've read a few posts (S.O., here) discussing migrating to `PrismApplication`, but no mentions of the new `PrismBootstrapper`. 

In addition, the tutorials only speak of `PrismApplication`. Now, it is much easier to just replace the base class from `UnityBootstrapper` to `PrismBootstrapper` and make a few adaptations than to move everything into the App.xaml.cs. 

Why was `PrismBoostrapper` introduced? Are there disavantages of using it as opposed to `PrismApplication`?

Thank you

**Comment by brianlagunas** [comment, marked as answer]
The bootstrapper is used for headless applications such as extensions, or custom implementations.  Just move to `PrismApplication`
